docs/self-managed/about-self-managed.md

```diff
@@ -7,29 +7,38 @@ title: "Camunda 8 Self-Managed"
 
 Camunda 8 Self-Managed is not Camunda 7. [Find Camunda 7 documentation here](https://docs.camunda.org).
 
-However, Optimize documentation is available for both Camunda 8 and Camunda 7, including deployment instructions available [here]($optimize$/self-managed/optimize-deployment/install-and-start). Look for `Camunda 7` badges to help you understand what content is available for what product.
-
 :::
 
-As an alternative to using Camunda 8 through SaaS, you can host it yourself; we call this setup Camunda 8 Self-Managed.
+Camunda 8 Self-Managed is a self-hosted alternative to using Camunda 8 SaaS. Building process automation solutions with Camunda 8 works similarly regardless of hosting and deployment.
 
-Building process automation solutions with Camunda 8 Self-Managed is similar to working with Camunda 8 SaaS. For more information on Camunda 8 SaaS, visit [What is Camunda 8?](../components/concepts/what-is-camunda-8.md) If you are new to Camunda 8, we recommend you start your journey with [Camunda 8 SaaS-based guides](../../guides/).
+For more information on Camunda 8 SaaS, visit [What is Camunda 8?](../components/concepts/what-is-camunda-8.md) If you are new to Camunda 8, we recommend you start your journey with [Camunda 8 SaaS-based guides](../../guides/).
 
 The content in this section of the documentation includes:
 
 - Everything you need to download, configure, and work with each component of Camunda 8 Self-Managed.
 - Features specific to Camunda 8 Self-Managed.
 
-The following components are available for Camunda 8 Self-Managed:
+To get started with your Self-Managed installation, see our [installation overview](./setup/overview.md).
+
+## Architecture
+
+Camunda 8 Self-Managed consists of multiple web applications and gRPC services. The following example architecture diagram uses two [Ingress objects](./setup/guides/ingress-setup.md#combined-ingress-setup) to access these services externally: an Ingress with HTTP(S) protocol for all web applications using a single domain, and another Ingress with gRPC protocol for the Zeebe workflow engine.
+
+![Camunda 8 Self-Managed Architecture Diagram - Combined Ingress](./assets/camunda-platform-8-self-managed-architecture-diagram-combined-ingress.png)
+
+:::note
+By default, the deployment requires at least three Kubernetes Nodes to meet the Pod Anti-Affinity constraints.
+:::
+
+In this configuration, Camunda 8 Self-Managed can be accessed as follows:
+
+- Web applications: `https://camunda.example.com/[identity|operate|optimize|tasklist|modeler]`
+  - Web Modeler also exposes a WebSocket endpoint on `https://camunda.example.com/modeler-ws`. This is only used by the application itself and not supposed to be accessed by users directly.
+- Keycloak authentication: `https://camunda.example.com/auth`
+- Zeebe gateway: `grpc://zeebe.camunda.example.com`
+
+## Availability
 
-- Zeebe
```

Comment:
I think this list is more appropriate on the install page, where we have to talk about how components can be installed in different configurations anyway, but could also see the argument for it being here instead (the goal being to only list it in one place instead of duplicating it). Ideally the About page would include more info about what & why to justify this content move.

Reply:
If we put this info in a reusable component, we could duplicate it without the maintenance overhead. Would that change your thoughts here?

Reply:
I've tried this iteration with a reused partial, which I think works nicely in both pages. If they start to diverge in content we can separate them out again.